_data/current.yml

```diff
@@ -1,3 +1,10 @@
+- name: TESTEXPO AND AGILE & DEVOPS EXPO 2020
```

Comment:
This conference is in the wrong place, it needs to be added based on the date. Can you move it to the right location?